src/rest_framework_api_key/permissions.py

```diff
@@ -8,6 +8,8 @@
 
 
 class KeyParser:
+    keyword = "Api-Key"
```

Comment:
Would it make sense to add a simple test that uses a custom `KeyParser.keyword` value and tests that that's what the library expects?

Reply:
Can we rename it to ‘authentication_type’ or ‘auth_type’ ?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization#syntax

Reply:
@florimondmanca wouldn't that type of test traditionally be in the code base that is actually doing the override? 

Otherwise there should be a test for every class testing overrides?

Reply:
@Mukhametvaleev i am okay with any naming scheme. However as @florimondmanca pointed out earlier, the django-rest-framework project uses the 'keyword' phrase.

Reply:
@brittandeyoung I believe what we want to test here is that users _can_ perform that override, since that's advertised in the docs and part of the public API now. Doing it ourselves once in a test is a nice way to ensure that, I think?

Reply:
This does make sense to test something we are suggesting users to do. I have added the tests. Please let me know what you think.